src/lib/personalTaxBreakdown.ts

```diff
@@ -156,9 +180,12 @@ function combineFederalAndProvincialForChart(
   const combined: { [key: string]: { federal: number; provincial: number } } =
     {};
 
-  // Add federal spending (excluding transfers)
+  // Add federal spending (excluding provincial transfers)
   federalSpending.forEach((category) => {
-    if (category.name !== "Transfer to Ontario") {
+    if (
```

Comment:
Does it make more sense to specify the federal transfer specified in the `PROVINCE_DATA` instead of doing this? This seems confusing